patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+
+All engineers in the organization can participate in the hackathon. The participants can be new to InnerSource, or InnerSource practitioners already. They can participate individually or as a team. Participating as a team also provides a safe environment, for example, those are new to InnerSource can team up InnerSource practitioners.
+
+There could be one or more categories to participate as follows:
+
+* Start a new InnerSource project: It could be either starting a new project from scratch as InnerSource or making a existing project InnerSource ready.
+* Contribute to the existing InnerSource project: InnerSource project owners can list down the features in GitHub issues as a pre-requisite before the hackathon and the participants can contribute to it during the hackathon.
+* Participate as an InnerSource mentor: Participants can also mentor new InnerSource practitioners and provide knowledge transfer or coaching during the event.
+
+The event could be held virtually or physically in different locations in the organization based on the cost and budget involved. A virtual event will have best ROI but it could depend on each organization.
+
+The winners and participants should be recognized and acknowledged in a company-wide forum at the end of the hackathon. This is important as it keeps motivating them and more engineers to adopt and practice InnerSource going forward.
+
+Such an event provides a safe space for the engineers who want to adopt or contribute to InnerSource but didn't have the time and motivation to do it or for those who kept deprioritizing due to high priority work deliverables. From middle management point of view, 1 or 2 days for such an event is not much of an ask and hence they are more likely to accept.
+
+## Resulting Context
+
+* There is increased adoption of InnerSource by providing such a safe platform to try and experiment combined with recognition and support by the senior leadership and middle management.
+* It results in more projects being made InnerSource ready and innersourced, and also results in more InnerSource contributions and start building community around existing InnerSource projects.
+* This happens not only during the event but continues after the event too, with the hackathon participants acting as InnerSource ambassadors in their teams.
+* It also helps ISPO and OSPO spread awareness about InnerSource best practices quickly, across the whole engineering community in the organization.
+* Such an event also helps discover some cool hobby projects that were developed to solve the needs of a specific team but turns out that many teams have similar requirements.
```

Comment:
I agree that "hobby" can be misleading. I like the re-phrase of @spier 

However the term "hobby" did make me think of a type of project that typically starts at or around Hackathons that I don't think of reading the rephrase.

What if we added:

* Such an event also helps find time to work on side projects that otherwise are dropped for lack of time or priority. Often those concern issues that all teams have to deal with. Impact locally though often is too low to address them - value only gets visible when looking at the bigger picture.

Think of the once famous Google 20% time that helped trigger GMail as legend has it. I'm trying to capture that as motivation of the Hackathon.